Library/ViewModels/PledgePaymentMethodsViewModelTests.swift

```diff
@@ -117,4 +118,35 @@ final class PledgePaymentMethodsViewModelTests: TestCase {
       self.notifyDelegateLoadPaymentMethodsError.assertDidNotEmitValue()
     }
   }
+
+  func testApplePayButton_applePayCapable_unsupportedProjectCountry_isHidden() {
```

Comment:
I think that the name `func testApplePayButton_isHidden_applePayCapable_unsupportedProjectCountry()` would describe better what we are testing here.